### PR TITLE
pkg/plugin: handle "AccessDeniedException" as user-induced

### PR DESCRIPTION
**Pull request by xdu31**
e.g.,

AccessDeniedException: The ciphertext refers to a customer master key that does not exist, does not exist in this region, or you are not allowed to access.

Signed-off-by: Gyuho Lee <leegyuho@amazon.com>

### CODE REVIEW

File: pkg/plugin/plugin_test.go [hunk 1]
```diff
@@ -91,6 +91,15 @@ func TestEncrypt(t *testing.T) {
 			healthErr: true,
 			checkErr:  true,
 		},
+		{
+			input:     plainMessage,
+			ctx:       nil,
+			output:    "",
+			err:       awserr.New("AccessDeniedException", "The ciphertext refers to a customer master key that does not exist, does not exist in this region, or you are not allowed to access", errors.New("fail")),
```
Comment (micahhausler) [inline pkg/plugin/plugin_test.go:98]: For completeness, can you add a test case for AccessDenied that is not user induced?

Comment (xdu31) [reply pkg/plugin/plugin_test.go:98]: > For completeness, can you add a test case for AccessDenied that is not user induced?

@micahhausler Added a test for the `AccessDenied` that is not user induced

**Review by micahhausler — APPROVED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**